index.css

```diff
@@ -0,0 +1,617 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+@font-face {
+  font-family: 'cocogoose';
+  src: url('./assets/fonts/cocogoose/cocogoose_pro_light-webfont.woff') format('woff');
+  src: url('/assets/cocogoose/fonts/cocogoose_pro_light-webfont.woff') format('woff2');
+  font-weight: normal;
+  font-style: normal;
+}
+
+*,
+::after,
+::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  line-height: 1.5;
+  font-size: 0.875rem;
+  min-height: 100vh;
+  max-width: 100vw;
+  margin: 0 auto !important;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+  font-size: 1rem;
+  color: #3e3e3e;
+}
+
+.cocogoose {
+  font-family: 'cocogoose', sans-serif;
+}
+
+.poppins {
+  font-family: 'poppins', sans-serif;
+}
+
+.container {
+  max-width: 100vw;
+  background-color: lightgray;
+}
+
+.logo {
+  display: none !important;
+}
+
+.hamburger {
+  background-color: transparent;
+  border: 0;
+  color: #3e3e3e;
+  cursor: pointer;
+  font-size: 42px;
+  right: 40px;
+  z-index: 10;
+  padding-left: 30px;
+}
+
+#close {
+  background-color: transparent;
+  border: 0;
+  color: #fff;
+  cursor: pointer;
+  font-size: 42px;
+  position: fixed;
+  right: 30px;
+  top: 20px;
+  z-index: 9;
+  width: 11px;
+  height: 11px;
+}
+
+.hide {
+  display: none;
+}
+
+.nav-ul {
+  display: none !important;
+}
+
+.nav-ul.show {
+  display: flex !important;
+  flex-direction: column !important;
+  align-items: center !important;
+  justify-content: center !important;
+  padding: 10% 8%;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  overflow: auto;
+  background-color: #3e3e3e;
+  backdrop-filter: blur(5px);
+}
+
+.left-margin {
+  margin-left: 0 !important;
+}
+
+.menu-text {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 32px;
+  color: #fff;
+}
+
+.page-title {
+  color: #ec5242;
+  font-size: 1.2rem;
+  letter-spacing: 2px;
+  line-height: 21px;
+  font-weight: 800;
+  padding-left: 10px;
+}
```

Comment:
- [ ] Great job with your design. Please make sure you increase the `font-size` of this section of your `About page`  to make it as close as possible to the Figma design🙏 

Original Design | Your About Page
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190467679-0578585e-7d02-4e84-8b06-73b50b5c18ec.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190467526-4dcbe276-cab0-498a-9dd4-2f9391191ee3.png)